Tella/Data/Database/UwaziDatabase.swift

```diff
@@ -0,0 +1,186 @@
+        }
+    }
+    func deleteUwaziTemplate(id: Int) {
+        do {
+            try statementBuilder.delete(tableName: D.tUwaziTemplate,
+                                    primarykeyValue: [KeyValue(key: D.cId, value: id)])
+        } catch let error {
+            debugLog(error)
+        }
+    }
+}
+// MARK: - Methods related to UwaziServerLanguageProtocol
+extension TellaDataBase: UwaziServerLanguageProtocol {
+    func createUwaziServerTable() {
+        let columns = [
+            cddl(D.cServerId, D.integer, primaryKey: true, autoIncrement: true),
+            cddl(D.cName, D.text),
+            cddl(D.cURL, D.text),
+            cddl(D.cUsername, D.text),
+            cddl(D.cPassword, D.text),
+            cddl(D.cAccessToken, D.text),
+            cddl(D.cLocale, D.text),
+        ]
+        
+        statementBuilder.createTable(tableName: D.tUwaziServer, columns: columns)
+    }
+    
+    func addUwaziServer(server: UwaziServer) -> Int? {
+        do {
+            let valuesToAdd = [KeyValue(key: D.cName, value: server.name),
+                               KeyValue(key: D.cURL, value: server.url),
+                               KeyValue(key: D.cUsername, value: server.username),
+                               KeyValue(key: D.cPassword, value: server.password ),
+                               KeyValue(key: D.cAccessToken, value: server.accessToken),
+                               KeyValue(key: D.cLocale, value: server.locale)
+            ]
+            
+            
+            return try statementBuilder.insertInto(tableName: D.tUwaziServer, keyValue: valuesToAdd)
+        } catch let error {
+            debugLog(error)
+            return nil
+        }
+    }
+    
+    func getUwaziServer(serverId: Int) throws -> UwaziServer? {
+        let response = try statementBuilder.selectQuery(tableName: D.tUwaziServer,
+                                                           andCondition: [KeyValue(key: D.cServerId, value: serverId)])
+        guard let uwaziServerDict = response.first else { return nil }
+        
+        return parseUwaziServer(dictionary: uwaziServerDict)
+        
+    }
+    
+    func parseUwaziServer(dictionary : [String:Any] ) -> UwaziServer {
+        let id = dictionary[D.cServerId] as? Int
+        let name = dictionary[D.cName] as? String
+        let url = dictionary[D.cURL] as? String
+        let username = dictionary[D.cUsername] as? String
+        let password = dictionary[D.cPassword] as? String
+        let token = dictionary[D.cAccessToken] as? String
+        let locale = dictionary[D.cLocale] as? String
+        return UwaziServer(id:id,
+                           name: name,
+                           serverURL: url,
+                           username: username,
+                           password: password,
+                           accessToken: token,
+                           locale: locale,
+                           serverType: ServerConnectionType(rawValue: 1)
```

Comment:
What do you think if you change the serverType in the `UwaziServer`  init implementation because already know the type